Explorer/Assets/DCL/PerformanceAndDiagnostics/Optimization/PerformanceBudgeting/Memory/MemoryBudget.cs

```diff
@@ -27,6 +27,16 @@ public class MemoryBudget : IMemoryUsageProvider, IPerformanceBudget
 
         private ulong actualSystemMemory => systemMemory.TotalSizeInMB;
 
+        public MemoryBudget(ISystemMemory systemMemory, IProfilingProvider profilingProvider) : this(
```

Comment:
As I can see this approach helps to familiarise with an class when it is seen at the first time. It's like default params that helps to see an allowed possible range, for example like:
`
public ExampleRequest(int attempts = 3, ulong timeoutMs = 10_000)
`

if an object is created with this constructor, it will still be a valid object that works according its contract